host/src/FunctionsNetHost/Native/NativeExports.cs

```diff
@@ -35,24 +35,24 @@ public static int GetApplicationProperties(NativeHostData nativeHostData)
                                                 delegate* unmanaged<byte**, int, IntPtr, IntPtr> requestCallback,
             IntPtr grpcHandler)
         {
-            Logger.LogDebug("NativeExports.RegisterCallbacks method invoked.");
-            
+            Logger.LogTrace("NativeExports.RegisterCallbacks method invoked.");
+
             try
             {
                 NativeHostApplication.Instance.SetCallbackHandles(requestCallback, grpcHandler);
                 return 1;
             }
             catch (Exception ex)
             {
-                Logger.LogInfo($"Error in RegisterCallbacks: {ex}");
+                Logger.Log($"Error in RegisterCallbacks: {ex}");
                 return 0;
             }
         }
 
         [UnmanagedCallersOnly(EntryPoint = "send_streaming_message")]
         public static unsafe int SendStreamingMessage(IntPtr pInProcessApplication, byte* streamingMessage, int streamingMessageSize)
         {
-            Logger.LogDebug($"NativeExports.SendStreamingMessage method invoked.");
+            Logger.LogTrace($"NativeExports.SendStreamingMessage method invoked.");
 
             try
             {
```

Comment:
Same comment here (and on all methods taking the handle) on using the handle vs the static approach.